utils.js

```diff
@@ -60,16 +61,20 @@ module.exports = {
      * @returns {string}
      */
     getDownloadUrl(revision) {
-        return `${this.getOsCdnUrl()}%2F${revision}%2F${this.getOsChromiumFolderName()}.zip?alt=media`;
+        let revisionPath = `/${revision}/${this.getOsChromiumFolderName()}`;
```

Comment:
Am I missing something or `getOsChromiumFolderName()` is an incorrect name of the function and it should be something like: `getOsChromiumBinaryName()`? It returns file name and not folder name, no?

Reply:
@dtolstyi I believe this function has existed since the initial commit of this repository years ago...

Reply:
That's true. But what I mean is if we should change it to proper name? 
It was probably me who created it long time ago and now I'm not very confident in subject area and I was hoping you are now more into it. So, I was wondering if it's actually **folder** or **file** name?

Reply:
Seems like it is used in two places, one is for a [folder name](https://github.com/ricksbrown/node-chromium/blob/60efb60fa1ad36a847d95235d0d830bac83abc03/utils.js#L38) and one is for a [file name](https://github.com/ricksbrown/node-chromium/blob/60efb60fa1ad36a847d95235d0d830bac83abc03/utils.js#L63). So it is named correctly for the first (and I think primary) scenario.

Reply:
Got it. Thank you!